rust/xaynet-analytics/src/database/screen_route/data_model.rs

```diff
@@ -0,0 +1,37 @@
+use chrono::{DateTime, Utc};
+use isar_core::object::{data_type::DataType, object_builder::ObjectBuilder};
+use std::vec::IntoIter;
+
+use crate::database::common::{FieldProperty, IsarAdapter};
+
+#[derive(Debug, PartialEq, Eq, Clone)]
+pub struct ScreenRoute {
+    pub name: String,
+    pub created_at: DateTime<Utc>,
```

Comment:
just the timestamp when the screen route was created. i can change the name to timestamp here, or change `AnalyticsEvent.timestamp` to `created_at` too, whichever makes most sense. Cause maybe in the future we want to track when it was deleted as well, at least for screen routes

Reply:
What's the difference between this and `AnalyticsEventType::ScreenEnter` ?

Reply:
`AnalyticsEventType::ScreenEnter` is the event that signals that the user entered a screen. A `ScreenRoute` identifies a unique screen, so that later in data combiner we can filter by `AnalyticsEvents` with the same `ScreenRoute`. does it make sense?